zds/pages/views.py

```diff
@@ -139,6 +142,111 @@ def alerts(request):
     })
 
 
+class CommentEditsHistory(ListView):
+    model = CommentEdit
+    context_object_name = 'edits'
+    template_name = 'pages/comment_edits_history.html'
+
+    def get_object(self):
+        return get_object_or_404(Comment, pk=self.kwargs['comment_pk'])
+
+    @method_decorator(login_required)
```

Comment:
Il y a le *mixin* `LoginRequiredMixin` qui est plus simple/propre à utiliser (https://docs.djangoproject.com/fr/1.10/topics/auth/default/#the-loginrequired-mixin)